temboardagent/plugins/maintenance/functions.py

```diff
@@ -435,3 +445,161 @@ def get_table(conn, schema, table):
     """
     conn.execute(query.format(schema, table))
     return dict(**next(conn.get_rows()))
+
+
+def schedule_vacuum(conn, database, schema, table, mode, datetimeutc, app):
+    # Schedule a vacuum statement through vacuum background worker
+
+    # Check that the specified table exists in schema
+    conn.execute(
+        "SELECT 1 FROM pg_tables WHERE tablename = '{table}' AND "
+        "schemaname = '{schema}'".format(table=table, schema=schema)
+    )
+    if not list(conn.get_rows()):
+        raise HTTPError(404, "Table %s.%s not found" % (schema, table))
+
+    # Schedule a new task to vacuum worker
+
+    # We need to build a uniq id for this task to avoid scheduling twice the
+    # same vacuum statement.
+    m = hashlib.md5()
+    m.update("{database}:{schema}:{table}:{datetime}".format(
+        database=database,
+        schema=schema,
+        table=table,
+        datetime=datetimeutc)
```

Comment:
``` python
m.update("{database}:{schema}:{table}:{datetime}".format(
    database=database,                                   
    schema=schema,                                       
    table=table,                                         
    datetime=datetimeutc).encode('utf-8')                
)                                          
```
otherwize we get the following error:
TypeError: Unicode-objects must be encoded before hashing